release/pypi/prep_binary_for_pypi.sh

```diff
@@ -95,6 +95,7 @@ for whl_file in "$@"; do
             fi
         )
 
+        rm -rf "${new_whl_file}"
```

Comment:
Why are we removing  this file before zipping ? I don't think this PR should modify the prep script